index.html

```diff
@@ -1,10 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Title</title>
-</head>
-<body>
-
-</body>
-</html>
\ No newline at end of file
+	<head>
+		<meta charset="UTF-8" />
+		<title>PR test</title>
+		<link rel="stylesheet" href="./styles.css" />
```

Comment:
I think this field should be before title

Reply:
Done